src/main/java/dukeexception/InvalidIndexException.java

```diff
@@ -0,0 +1,14 @@
+package dukeexception;
+
+public class InvalidIndexException extends DukeException {
+
+    /**
+     * Return message of the Thrown InvalidIndexException.
+     *
+     * @return String to be printed.
+     */
+    @Override
+    public String getMessage() {
```

Comment:
(this comment applies for every Exception, apart from `DukeException`)

Interesting way to customise each Exception. I think it would be slightly better if the message was passed to the super constructor though, so other methods in `Exception` like `getLocalizedMessage` would also display the custom message

Example (simplified):
```
public class InvalidIndexException extends Exception {
    InvalidIndexException() {
        super("ERROR: INVALID INDEX");
    }
}
```